src/components/entity/ha-chart-base.html

```diff
@@ -0,0 +1,182 @@
+<link rel='import' href='../../../bower_components/polymer/polymer.html'>
+
+<dom-module id="ha-chart-base">
+  <link rel="import" type="css" href="../../../bower_components/billboard.js/dist/billboard.min.css">
+  <style>
+     :host {
+      display: block;
+      /* @apply --my-custom-chart-istic-mixin; */
+    }
+  </style>
+  <template>
+    <div id="chartTarget"></div>
+  </template>
+</dom-module>
+<script src="../../../bower_components/d3/d3.js"></script>
```

Comment:
Note to self to not merge this until I have reworked this to be loaded on demand.